Source/CarthageKit/Xcode.swift

```diff
@@ -992,6 +995,15 @@ public func buildInDirectory(directoryURL: NSURL, withConfiguration configuratio
 							|> map { _ in scheme }
 					}
 					|> collect
+					|> catch { error in
+						switch error {
+						case .NoSharedSchemes:
+							return .empty
+
+						default:
+							return SignalProducer(error: error)
+						}
+					}
```

Comment:
Do you mean that this lets Carthage continue to run after finding a project with no shared dependencies?